Documentation/ceph-pool-crd.md

```diff
@@ -80,6 +80,13 @@ spec:
   mirroring:
     enabled: true
     mode: image
+    # schedule(s) of snapshot
+    snapshotSchedules:
+      - interval: 24h # daily snapshots
+        startTime: 14:00:00-05:00
+      # another schedule
```

Comment:
seems fine to just have a single example instead of a comment that is the same, if you want to remove it